cilium/cmd/cleanup.go

```diff
@@ -133,6 +134,21 @@ func removeCNI() error {
 	return err
 }
 
+func unmountCgroup() error {
+	cgroupRoot := defaults.DefaultCgroupRoot
+	cgroupRootStat, err := os.Stat(cgroupRoot)
+	if err != nil {
+		return fmt.Errorf("Failed to stat the mount path %s: %s", cgroupRoot, err)
+	} else if !cgroupRootStat.IsDir() {
+		return fmt.Errorf("%s is a file which is not a directory", cgroupRoot)
+	}
+
+	if err := syscall.Unmount(cgroupRoot, syscall.MNT_FORCE); err != nil {
```

Comment:
I think that we should add an Info level log message to state that this is being unmounted, so that the user can become aware that this may conflict with other configurations of the CGroup.

Reply:
I see a `showWhatWillBeRemoved()` function which seems to outline the changes that this tool will make. Perhaps a better way to signal that CGroup unmount will also occur would be to also add a line to the output printed by that function.

Reply:
@joestringer this should be safe even with other configurations of cgroupv2. We are allowed to have multiple mount points here so unmounting the Cilium mount point should be fine.